### ISSUE TITLE
Is the Page dependency in Accessibility domain missing?

### ISSUE THREAD

**Issue by sanfusu** [ChromeDevTools/devtools-protocol#256]
https://github.com/ChromeDevTools/devtools-protocol/blob/87addc3fb7af1c1c98e476f0fdc4bc5011815ac0/json/browser_protocol.json#L10

The Page is referred in here: 
https://github.com/ChromeDevTools/devtools-protocol/blob/87addc3fb7af1c1c98e476f0fdc4bc5011815ac0/json/browser_protocol.json#L336
